app/server-utils/url-to-iframe.spec.ts

```diff
@@ -0,0 +1,14 @@
+import { urlToIframe } from "./url-to-iframe"
+
+describe('urlToIframe', () => {
+  it('should not convert an anchor tag whose href does not match tag content', () => {
+    const input = `<a href="https://aisafety.world/">this map</a>`
+    expect(urlToIframe(input)).toBe(input);
+  });
+
+  it('should convert an anchor tag whose href matches tag content', () => {
+    const input = `<a href="https://aisafety.world/">https://aisafety.world/</a>`
+    const output = `<iframe src="https://aisafety.world/" sandbox="allow-scripts allow-same-origin"/>`
+    expect(urlToIframe(input)).toBe(output);
+  });
+})
```

Comment:
did you run prettier for the new files (`npm run lint`)? I believe it should have added newline at the end of files automatically...

otherwise I like the first test 👍

Reply:
Thanks, yes, you were right, it needed a prettier run https://github.com/StampyAI/stampy-ui/pull/193/commits/bbb7a85290548f2b6f459252704e1b55ae1efa26